backend/resources/plan.py

```diff
@@ -134,5 +134,5 @@ def delete(self, id):
         except (PlanCreationError, UserCreationError, PlanDeletingError) as e:
             return {'message': e.message}, 400
         except Exception as e:
+            print(e)
```

Comment:
this line should be deleted after testing 😸

Reply:
Fixed! Thank u